src/mlpack/core/data/load_csv.hpp

```diff
@@ -1,8 +1,28 @@
 /**
  * @file core/data/load_csv.hpp
  * @author ThamNgapWei
+ * @author Conrad Sanderson
+ * @author Gopi M. Tatiraju
  *
- * This is a csv parsers which use to parse the csv file format
+ * This csv parser is designed by taking reference from armadillo's csv parser.
+ * In this mlpack's version, all the arma dependencies were removed or replaced
+ * accordingly, making the parser totally independent of armadillo.
+ *
+ * https://gitlab.com/conradsnicta/armadillo-code/-/blob/10.5.x/include/armadillo_bits/diskio_meat.hpp
```

Comment:
```suggestion
 * https://gitlab.com/conradsnicta/armadillo-code/-/blob/10.5.x/include/armadillo_bits/diskio_meat.hpp
 *
 * The original Armadillo parser is licensed under the BSD-compatible Apache license, shown below:
 *
```
I think it could be useful just to point out why there are two licenses in the header file.